packages/wrangler/src/entry.ts

```diff
@@ -15,7 +16,12 @@ import type { Metafile } from "esbuild";
  *
  * It consists not just of a `file`, but also of a `directory` that is used to resolve relative paths.
  */
-export type Entry = { file: string; directory: string; format: CfScriptFormat };
+export type Entry = {
+	file: string;
+	directory: string;
+	format: CfScriptFormat;
+	moduleRoot: string;
```

Comment:
I think we need to start adding JSDoc comments to these properties.

It is not clear what the difference is between `directory` and `moduleRoot`. So it would be helpful to describe them here.

In fact I wonder if "module" root is really the right name? I am probably wrong but it feels like it should be something like "external module root" or "additional module root" or something?

But perhaps a doc comment would be enough to cover that...

Reply:
I think module root captures the intent (it's not really 'additional' or 'external', since it includes all module except the entrypoint), but I've added some doc comments